Traceback (most recent call last):
  File "C:\Users\adnegrau\AppData\Local\Programs\Python\Python312\Lib\site-packages\jupyter_cache\executors\utils.py", line 58, in single_nb_execution
    executenb(
  File "C:\Users\adnegrau\AppData\Local\Programs\Python\Python312\Lib\site-packages\nbclient\client.py", line 1305, in execute
    return NotebookClient(nb=nb, resources=resources, km=km, **kwargs).execute()
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\adnegrau\AppData\Local\Programs\Python\Python312\Lib\site-packages\jupyter_core\utils\__init__.py", line 165, in wrapped
    return loop.run_until_complete(inner)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\adnegrau\AppData\Local\Programs\Python\Python312\Lib\asyncio\base_events.py", line 685, in run_until_complete
    return future.result()
           ^^^^^^^^^^^^^^^
  File "C:\Users\adnegrau\AppData\Local\Programs\Python\Python312\Lib\site-packages\nbclient\client.py", line 705, in async_execute
    await self.async_execute_cell(
  File "C:\Users\adnegrau\AppData\Local\Programs\Python\Python312\Lib\site-packages\nbclient\client.py", line 1058, in async_execute_cell
    await self._check_raise_for_error(cell, cell_index, exec_reply)
  File "C:\Users\adnegrau\AppData\Local\Programs\Python\Python312\Lib\site-packages\nbclient\client.py", line 914, in _check_raise_for_error
    raise CellExecutionError.from_cell_and_msg(cell, exec_reply_content)
nbclient.exceptions.CellExecutionError: An error occurred while executing the following cell:
------------------
import json
import os
# ROOT_DIR = os.path.dirname(os.path.abspath(__file__)) # This is your Project Root

# cwd = os.getcwd()
# print(cwd)
totalNumberOfDocuments = 200
chunk_size = 300
chunk_overlap = 0
file_path = f"../output/chunks-solution-ops-embedded-{totalNumberOfDocuments}-{chunk_size}-{chunk_overlap}.json"

if(not os.path.exists(file_path)):
    file_path = f"./output/chunks-solution-ops-embedded-{totalNumberOfDocuments}-{chunk_size}-{chunk_overlap}.json"
with open(file_path) as f:
    data = json.load(f)
    dfcluster = pd.DataFrame(rec for rec in data)
------------------


[1;31m---------------------------------------------------------------------------[0m
[1;31mFileNotFoundError[0m                         Traceback (most recent call last)
Cell [1;32mIn[3], line 14[0m
[0;32m     12[0m [38;5;28;01mif[39;00m([38;5;129;01mnot[39;00m os[38;5;241m.[39mpath[38;5;241m.[39mexists(file_path)):
[0;32m     13[0m     file_path [38;5;241m=[39m [38;5;124mf[39m[38;5;124m"[39m[38;5;124m./output/chunks-solution-ops-embedded-[39m[38;5;132;01m{[39;00mtotalNumberOfDocuments[38;5;132;01m}[39;00m[38;5;124m-[39m[38;5;132;01m{[39;00mchunk_size[38;5;132;01m}[39;00m[38;5;124m-[39m[38;5;132;01m{[39;00mchunk_overlap[38;5;132;01m}[39;00m[38;5;124m.json[39m[38;5;124m"[39m
[1;32m---> 14[0m [38;5;28;01mwith[39;00m [38;5;28;43mopen[39;49m[43m([49m[43mfile_path[49m[43m)[49m [38;5;28;01mas[39;00m f:
[0;32m     15[0m     data [38;5;241m=[39m json[38;5;241m.[39mload(f)
[0;32m     16[0m     dfcluster [38;5;241m=[39m pd[38;5;241m.[39mDataFrame(rec [38;5;28;01mfor[39;00m rec [38;5;129;01min[39;00m data)

File [1;32m~\AppData\Local\Programs\Python\Python312\Lib\site-packages\IPython\core\interactiveshell.py:310[0m, in [0;36m_modified_open[1;34m(file, *args, **kwargs)[0m
[0;32m    303[0m [38;5;28;01mif[39;00m file [38;5;129;01min[39;00m {[38;5;241m0[39m, [38;5;241m1[39m, [38;5;241m2[39m}:
[0;32m    304[0m     [38;5;28;01mraise[39;00m [38;5;167;01mValueError[39;00m(
[0;32m    305[0m         [38;5;124mf[39m[38;5;124m"[39m[38;5;124mIPython won[39m[38;5;124m'[39m[38;5;124mt let you open fd=[39m[38;5;132;01m{[39;00mfile[38;5;132;01m}[39;00m[38;5;124m by default [39m[38;5;124m"[39m
[0;32m    306[0m         [38;5;124m"[39m[38;5;124mas it is likely to crash IPython. If you know what you are doing, [39m[38;5;124m"[39m
[0;32m    307[0m         [38;5;124m"[39m[38;5;124myou can use builtins[39m[38;5;124m'[39m[38;5;124m open.[39m[38;5;124m"[39m
[0;32m    308[0m     )
[1;32m--> 310[0m [38;5;28;01mreturn[39;00m [43mio_open[49m[43m([49m[43mfile[49m[43m,[49m[43m [49m[38;5;241;43m*[39;49m[43margs[49m[43m,[49m[43m [49m[38;5;241;43m*[39;49m[38;5;241;43m*[39;49m[43mkwargs[49m[43m)[49m

[1;31mFileNotFoundError[0m: [Errno 2] No such file or directory: './output/chunks-solution-ops-embedded-200-300-0.json'

